src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -440,6 +440,7 @@ public Page Pop()
             else
             {
                 Remove(curTop);
+                curTop.Dispose();
```

Comment:
When we dispose page here, it will return `Disposed` page.
I think we need to find other good way ;ㅅ;

Reply:
Agreed.
as far as I know, pop is only intent to pop out the page on navigator, so disposing or keepping it and reusing is application's choice. if we might provide RemovePage() or DeletePage() in Navigator, we could process Dispose() internally as it will not return the page and API intended to remove page object.
please consider remove this line or adding other APIs with discussing @Jaehyun-Cho

Reply:
Thank you for your reviews.
I added `Hide()` as you mentioned here at first, but `RemovePopup()` cannot be called because `Page` is not AccessibilityStates.Modal. And I thought I should match Init and Dispose.
However, as you told, I'll think about something else.

Reply:
Hello all,
I made 'DeletePage()' to call Dispose() internally. Could you check it again?

Reply:
hmm.. what I meant was Pop() should only processing "pop" action without delete any objects... so if we provide another public API DeletePopup() instead of calling Pop(). or they can dispose popped object which is currently provided.
To solve the problem, how about manaing accessibility list instead of disposing object?
adding accessibility tree when it is added in the navi and remove tree when it popped.

Reply:
Oh I see. I added to call Show/Hide directly instead of adding accessibility tree.
What do you think?

Reply:
So if object is hided, object will be disqualified accesibility highlight candidate? that seems good solution that I look. 👍 
@jaehyun0cho have you other opinion about this?

Reply:
I think that it needs to move the APIs to the correct places.
I left my comments on RemovePageToList() and AddPageToList() below.